README.md

```diff
@@ -1,32 +1,28 @@
 # System **MyCanteen**
 
-## Krótki opis projektu
-
-Stworzone oprogramowanie ma być odpowiedzialne za zarządzanie ruchem w stołówkach, ich opiniowaniem, usprawnianiem pracy pracowników w stołówkach oraz ułatwieniem klientom stołówek pozyskiwania zarówno informacji dotyczących aktualnego stanu stołówek, jak i zasięgania opinii innych użytkowników dotyczących wszelakich potraw serwowanych w stołówkach.
-
-## Funkcjonalności stworzonego oprogramowania
-
-Stworzony system ma w przyjazny dla użytkowników sposób umożliwiać im:
-
-1. Śledzenie aktualnego stanu stołówek:
-    * zbieranie informacji na temat aktualnego zatłoczenia w pomieszczeniach stołówki z wykorzystaniem kamer internetowych
-    * zbieranie informacji na temat aktulanego menu w stołówkach od użytkowników (rozpoznawanie tekstu ze zdjęć menu) jak i możliwość dodawania aktualnego menu przez osoby prowadzące lokale
-    * zbieranie informacji na temat aktualnych długości kolejek w lokalach, średniego czasu oczekiwania na posiłek
-2. Zbieranie danych od użytkowników aplikacji:
-    * wystawianie opinii na temat posiłków serwowanych w stołówkach każdego dnia
+System MyCanteen jest przeznaczony dla klientów oraz właścicieli stołówek. Klienci przed wyjściem na obiad będą mogli sprawdzić aktualne obłożenie stołówki, oferowane menu, a także ocenić serwowane potrawy. Właściciel stołówki może przeglądać zebrane informacje, zapoznać się z historycznymi trendami odwiedzin, a także przeglądać informacje zwrotne.
+
+## Zakres funkcjonalny
+
+1. Śledzenie liczby użytkowników:
+    * zbieranie informacji o aktualnym zatłoczeniu stołówki
+    * zbieranie informacji o aktualnych długościach kolejek i średnim czasie oczekiwania na posiłek
+    * podgląd informacji o obłożeniu na żywo
+    * przegląd historii i trendów dla właścicieli
+        * powiadomienia o spodziewanych skokach liczby użytkowników
+        * szacowanie ilości zamówionych dań 
```

Comment:
Mam wątpliwości, czy to się da zrobić. Chyba że płyniemy z pomysłem @avan1235? :)

Reply:
@kantoniak który punkt zdaje się nierobialny? Podląd i przegląd mam wrażenie że będą robialne, te powiadomienia o skokach nie wiem tylko czym mogłyby być powodowane, bo z tego co rozumiem to system nie ma wykraczać poza stołówki. Szacowania można próbować dokonywać na podstawie historycznych danych, bo nazwy potraw zdają się być powtarzalne (w sensie jakoś się wszędzie zapętlają serwowane menu)

Reply:
Tyle tylko, że nie wiemy kto co zamawiał. Cały nasz input to liczba osób wchodzących i wychodzących, nie da się nawet ustalić kto jest w środku (i to chyba dobrze przy założeniach odnośnie prywatności).

Reply:
tak, ale np. PAni przy kasie jak wystawia rachunek mogłaby odklikiwac co bylo kupione. Brak powiazania z nazwiskiem, a dane dotyczce tego, jakie potrawy sa lubiane moga byc przydatne

Reply:
Nikt by się nie zgodził na to, żeby jeszcze coś dodatkowo odklikiwać. Przykładowo w Kubusiu ludzie którzy odebrali obiad stoją w kolejce, bo płatność za długo trwa.

Gdyby to się dało robić automatycznie to by było spokor, ale raczej nie będziemy rozszerzać funkcjonalności kas fiskalnych.